pkg/gui/gui_common.go

```diff
@@ -157,6 +157,14 @@ func (self *guiCommon) State() types.IStateAccessor {
 	return self.gui.stateAccessor
 }
 
+func (self *guiCommon) IgnoringWhitespaceSubTitle() string {
```

Comment:
I want `gui_common` to only hold stuff that could be of general use to avoid namespace pollution, and I don't think this qualifies. I'd chuck this in `pkg/gui/controllers/helpers/diff_helper.go` instead.

Reply:
Yeah, I was unsure if this is a good place, but then forgot to ask. Moved to `diff_helper.go` in 0ae613290536.